hdat/__init__.py

```diff
@@ -0,0 +1 @@
+from .suite import Suite, MetricsChecker # noqa
```

Comment:
What does `noqa` mean?

Also, do you think we should not split the metrics checker into a separate file? https://github.com/innolitics/hdat/issues/18

Reply:
Ahh, is it for flake? https://github.com/PyCQA/pycodestyle/issues/476

Reply:
Yeah, flake complains that you're importing stuff but not using it. 

I forgot to go over the module issue. I'm pretty sure it's OK to group tightly coupled classes together in the same module. The way I'm imagining it, you would only be using MetricsChecker for suite.check, and suite.check would almost always use a checker object. It would make more sense to split them if they're going to be used separately.

Reply:
Yeah, I think you are right about the coupling.  The MetricsChecker's API is tightly coupled to the Suite.check API.  Good point.

Everything looks good!